src/dom/Struct.ts

```diff
@@ -59,6 +59,13 @@ export class Struct extends Value(Object, IonTypes.STRUCT, FromJsConstructor.NON
                     return target[name];
                 }
                 return target._fields[name];
+            },
+            deleteProperty: function (target, name): boolean {
+                // Property is deleted only if it's in _field Collection
+                if (name in target._fields) {
+                    delete target._fields[name];
+                }
+                return true;
```

Comment:
Because we had to do some research to understand what `true` means in the context of the `delete` operator, I suggest adding a comment with a link to [the relevant documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete#Return_value).